server/hydrogen-render/1-render-hydrogen-to-string.js

```diff
@@ -84,6 +89,11 @@ async function renderHydrogenToString(options) {
       });
     });
 
+    assert(
+      data,
+      `No HTML sent from child process to render Hydrogen. Any child errors? (${childErrors.length})`
```

Comment:
We don't expect any `childErrors` if this is ever encountered but is just a sanity check. We don't even give the relevant info if there was `childErrors` 🤷